airbyte-integrations/connectors/source-bigquery/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,19 @@
+{
+    "streams": [
+      {
+        "stream": {
+          "name": "airbyte_tests_compr_otzwknbj/airbyte_tests_compr_otzwknbj_1_bigint",
+          "json_schema": {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "type": "object",
+            "properties": {}
+          },
+          "supported_sync_modes": ["full_refresh", "incremental"],
+          "source_defined_cursor": false,
+          "default_cursor_field": ["column_name"]
```

Comment:
This seems like an odd cursor field. Where does this come from?

Reply:
since I wasn't using it before, I didn't update it. It's updated now to used the correct cursor field